globalConfig.json

```diff
@@ -134,7 +134,7 @@
                             "field": "dataset_log_read_access_key",
                             "label": "DataSet Log Read Access Key",
                             "type": "text",
-                            "help": "Required to enable inputs and SPL comand. Include trailing hyphens if applicable.",
+                            "help": "Required to enable inputs and SPL comand. Include trailing hyphens if applicable. Please ignore this if AuthN token value is provided.",
```

Comment:
I suppose same should go to `dataset_log_write_access_key` right?

I would prefer eg. `Required (if no AuthN token provided) to enable inputs and SPL comand. Include trailing hyphens if applicable.`

Reply:
No, write key is different, this only applicable to read ones only